data/participants.json

```diff
@@ -448,5 +448,15 @@
             "twitter"  : "",
             "referral" : ""
 	    }
```

Comment:
The syntax is not correct. Check the validity using [this](https://jsonlint.com/)